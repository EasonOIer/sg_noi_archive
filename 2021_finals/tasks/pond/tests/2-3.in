1953 1558
726515 77467 64439 528348 192980 28454 455192 888883 736710 992302 306004 594934 132203 627553 939882 297527 58146 578610 688603 703941 297167 453202 918314 256499 97616 639986 673238 49680 659080 105794 645410 170927 93271 962931 677709 37599 95646 601399 289241 294899 390079 662079 251335 367355 66970 790139 777879 981192 48208 290013 685964 434283 496643 652182 698469 685557 372627 346262 262948 500599 778977 526350 605770 242947 878095 298353 564632 48808 158269 869357 294222 798648 723212 750774 619246 645317 188190 339323 68218 42207 348435 508013 838193 4630 13133 213502 814805 422077 644267 58143 771110 239204 193232 850624 530663 925212 396572 841939 480663 473014 816433 345992 495928 562563 782383 763793 789515 275924 663927 921183 705553 632250 111723 344610 798091 753670 685291 444491 599161 467146 897368 711309 714014 926677 579412 407703 612665 918522 163583 92186 603276 986440 963578 394504 151192 762770 230055 702290 123933 273960 365542 494696 576478 251098 167192 500604 266522 76155 431070 327294 671699 745691 477466 975136 945482 424907 542573 794450 68928 377117 195382 831750 4854 299296 132376 386477 824044 761318 122227 201591 298788 155840 824553 856278 164447 923328 935578 322810 832094 939189 283111 749781 837026 541682 940530 662360 973484 944135 951514 825075 882876 33092 758339 486872 828692 571388 105547 207031 763125 123583 561237 730050 96672 525278 16065 426473 347798 58730 874353 61150 639511 165045 869828 355282 112678 529998 826078 483253 918274 272647 903133 733049 570861 345521 269028 920782 600680 907806 549651 444838 897268 759325 16908 641457 129553 615809 190190 300961 618305 907098 467164 649959 365062 148537 400119 692979 989151 929006 513759 436407 924557 460 583157 146723 768083 338787 3290 26921 923185 525700 715952 27450 248231 788737 366231 322438 986231 16041 312561 463166 918434 371406 601821 209493 258601 388162 841150 52281 449804 166292 260405 38110 830990 568791 812514 258611 676654 189146 741158 354281 111662 859376 861282 784051 832335 244457 770296 659391 359122 79830 231560 372033 70584 741256 349235 836455 359553 916624 967323 375845 720175 158463 363616 174826 425795 220582 841123 609575 659710 993430 500230 845455 962593 158753 993641 964496 471916 350550 582957 79838 554210 354952 912820 761128 66370 185862 475806 336927 717134 870435 147468 550133 17223 714095 848032 531035 823598 946362 257722 37057 222851 904535 44155 572697 113737 770774 724997 350163 690111 108232 708344 74209 910183 873918 622773 355700 695426 550858 141278 500412 482896 869819 802323 372907 791764 814115 881525 969905 570829 551956 504806 669376 745661 993501 421958 156296 927077 677585 894070 901307 228614 918975 402008 772106 514465 369018 594828 66981 876982 569682 458867 785790 179911 857176 801902 990213 584662 758749 786463 136172 297058 311922 493543 547363 833260 615336 212837 369244 619767 492432 402492 632904 128322 266364 600844 236375 399632 267771 255223 820987 182048 258656 371229 467620 478900 317498 467295 29060 999628 795420 554526 109068 185387 530773 352188 588972 668624 342456 707673 446486 788670 271317 710423 390522 88628 811581 512534 48842 116 303617 47799 28918 667902 350066 120028 980208 926229 55265 661668 755753 551672 303711 814067 154897 939241 298622 692775 866522 858217 464333 11555 975348 899641 318751 615687 601617 642202 150083 541300 311905 815756 210188 535372 271318 880914 142793 269295 73441 441195 6285 5247 89522 757081 89576 48540 257083 774365 8811 225511 33004 545399 495911 452289 3575 518058 877869 157772 226509 506229 870606 314152 187998 47386 252316 57974 7004 297754 151910 19928 754167 434870 193598 280686 779041 979119 853750 902746 236682 448901 185757 779579 869732 44347 2694 973813 845280 578256 254529 425446 150318 461958 690335 145201 151167 606045 979120 823158 614406 707928 449735 613526 458034 124470 585716 332211 396532 78394 128840 302829 103102 324771 617813 701925 118337 798899 708650 294589 717613 660842 858521 264213 123060 81803 20276 882702 225798 523093 654218 686090 721813 707082 236974 275497 902161 968876 785071 419212 111404 845817 541984 789589 958743 248821 16529 63460 584834 388192 444307 969114 636831 433722 641911 657561 748382 107397 827598 282049 40305 847618 964713 194419 603721 610546 909657 319889 755798 668822 276528 153723 341514 942446 666685 297490 183317 558525 589458 865464 116073 409154 967681 118989 754550 196031 909337 936050 91758 611124 442925 45384 739458 932004 884429 145969 566050 456655 53898 608654 662464 151861 599541 430398 780812 398296 702495 19044 582712 883449 956761 887904 274869 820761 915380 201748 628019 817784 226177 18976 554851 583321 993105 712101 104503 325371 832822 40517 477600 769824 251455 770144 412943 909641 244607 909079 557620 360823 53573 939083 341248 222250 425700 841118 980517 896101 156204 452208 158804 475239 801454 613923 121085 15666 406971 677079 87506 808967 543332 215597 33667 250791 121210 611654 699778 263186 99470 808132 227840 810827 9457 798023 967149 663569 703924 675825 426875 143334 345705 724194 282586 531659 650079 947579 241276 973696 811625 302237 581969 366472 160726 267946 933580 318882 197113 429506 983259 761552 736546 368007 597425 563731 576930 482329 241663 383596 615050 803097 224565 506268 722987 66728 200133 299811 94358 156392 45024 140356 215068 411129 895030 175230 344718 705788 331577 110547 186814 643091 115801 855492 871547 565342 633480 150929 135639 919551 825910 625341 310628 93077 955477 820999 95218 388355 314390 183985 359633 61482 208553 814716 311252 339882 403776 492073 321995 27556 806859 262456 734983 115641 912892 237484 287193 229323 464387 14916 909650 463418 364626 612802 412163 754791 965146 49127 533438 576756 995467 812468 115218 937217 448919 222185 90513 968111 181806 389257 849357 725152 165378 708883 196757 998708 655811 310537 926076 101580 614564 510892 350994 525002 155539 46780 329448 320782 414595 148220 303951 140153 534181 853227 706648 299949 629482 111244 955796 279306 703170 664606 15444 346097 801822 982498 764649 978052 747504 727515 434148 262755 405033 444681 732759 160303 141177 573292 820497 692746 521106 803974 672997 424084 638244 95832 594210 534830 627169 530979 971227 418361 242234 664935 674080 479255 949663 800487 308865 205225 114242 940514 236369 622542 699091 186458 7136 239465 467652 375312 538367 48154 460678 886533 114623 835 276041 186362 654898 835780 209662 383023 700903 972414 919256 477670 672112 647699 294725 911329 426207 818082 546681 277342 753219 583887 472035 300321 615215 617528 495540 139946 739699 556224 677342 290681 475679 337543 349202 550928 493766 41455 446656 925424 200760 183320 903450 885466 952070 165539 988497 952975 75649 66545 543573 158672 72250 784612 794839 470685 147139 801244 112649 929976 957227 691067 582108 764898 336708 990177 189504 495452 86176 714671 775015 44711 477518 325510 507714 467459 513889 613320 578689 415731 216169 205973 484501 946486 877623 190543 402352 585294 29083 968149 939267 511612 520104 259851 810239 754496 954414 998287 85204 115549 504446 355332 782259 982610 435498 325194 346574 134817 155329 780610 638998 687499 403669 262286 915899 696593 637944 826190 648367 812276 219833 149604 990242 585652 743787 527599 588358 628179 6365 700498 230156 315410 626914 68526 72036 375602 97802 884386 400896 290420 915878 795665 38224 112537 484999 677980 612230 545633 328446 328035 709010 973838 171612 490247 992166 788641 569522 219625 164979 555291 217809 121323 131177 142781 445259 650669 951674 416328 898799 630096 946518 950588 53648 602422 58991 458395 322173 201599 742774 727078 92617 163212 334235 787102 519555 626687 499791 523460 232126 666360 637125 680536 918026 451992 620011 410525 861112 158977 524917 168850 991385 764435 255250 500473 744766 94255 740915 651347 995641 236151 847114 441696 801817 249675 629000 299875 901552 978188 545757 354267 52560 213467 577019 663754 519368 611802 194003 320561 109291 915818 992469 585233 697928 891178 553714 746027 505650 481405 115356 198015 109177 983619 568644 939344 197870 508126 745050 173213 347618 423086 580688 723560 757542 829332 298775 191922 109195 738610 265567 758267 518946 895144 128794 647686 146749 648087 195704 616082 464491 681888 626091 922145 299326 967205 658595 356596 141320 726727 320146 213004 455897 73381 830056 486441 329217 552226 679447 972622 847379 161566 930192 902330 17631 371387 432579 783727 998902 540760 552738 390638 830313 744729 62750 23095 98761 621276 406439 47181 302823 793348 611118 483439 249620 598321 413925 810912 936866 310917 9853 917023 501472 606047 832678 73773 322400 658787 823058 65613 721588 103023 710770 658431 812012 627558 109686 962499 86616 83624 939279 673499 536075 545303 423792 943432 977365 659026 784327 496011 853703 410044 23560 80997 464818 878006 481122 458326 681895 369248 856134 556128 467639 695579 711974 628357 950451 710088 863731 527236 512877 788681 355293 596254 623860 877438 758166 51926 229147 342856 181774 749355 95985 308553 251895 133856 793417 396641 398594 585972 72755 461682 523551 587241 10303 972699 239488 816704 171904 314495 752405 1273 675206 703551 250000 379543 232738 804602 201569 994649 591013 496724 775482 958785 711457 845099 847756 259984 914789 515746 713543 410118 162947 993305 759645 333606 744205 505329 213886 835439 854546 890380 991516 350500 251109 246154 404840 24516 227349 119052 92754 261841 878325 757108 40715 792331 925397 100194 302854 830837 787782 840439 239985 82318 266585 502421 830720 926296 411167 640348 398818 62932 539305 560365 774984 752323 755592 248425 69605 262182 251085 873970 359079 830333 517386 66665 783553 476555 318350 766381 382180 320399 161068 320097 333121 836107 722355 182886 985352 467776 711798 21211 836458 498990 86853 805411 400474 682072 494513 999838 219782 513085 884491 248302 951458 940383 991588 400563 987420 477311 944909 335438 320552 575582 180631 391777 822005 824445 819795 615116 942005 604060 49170 818480 532248 941437 184166 514670 101520 430625 379038 385576 958925 130953 594573 296574 231745 147958 503106 646802 86621 947497 370576 702020 51025 652322 394183 368245 319950 868259 82104 209031 215623 672493 342451 280793 680042 623745 619403 960015 853547 939232 78417 446599 158649 640266 558932 398500 344645 727554 695334 480198 250562 992199 37784 575055 928670 978075 130940 354590 664333 186047 155297 61399 490557 704500 816986 319192 69474 810874 828342 665796 590763 119059 163656 674491 639882 974145 792480 333982 161711 791107 826255 934671 882124 731546 227942 268718 267537 438025 979638 898525 202096 938896 263376 180412 671065 208047 43845 626220 660670 761000 455654 905358 951056 333244 892868 100676 918911 174700 262385 362606 648637 950520 774268 620909 690235 471523 481912 750115 317673 428268 295285 612016 286968 414997 801027 395189 642394 38303 225013 621268 822592 135115 226651 589677 883930 274393 330182 570786 890522 94348 614727 862546 685233 657955 72047 499168 803215 154656 354319 91565 158181 769796 729529 292824 515813 494096 695804 447428 81920 572292 844074 13202 636966 190051 664597 735074 704777 329068 748684 972501 191130 56801 354850 671308 181065 742531 499687 648617 801608 644179 619628 502028 426887 390097 338241 610198 492408 562624 470841 837096 892683 270303 907684 43748 319666 861424 930452 606507 703070 44980 769944 909724 246975 955435 870658 441690 927238 406360 114339 118342 443430 446906 503181 432191 581453 789011 170575 225256 152431 176548 667004 764897 517179 469614 924068 405452 580899 947122 556951 863817 377495 979376 610984 83129 751944 281898 96551 504318 905769 389794 761573 704696 758825 240343 980275 3528 711801 400970 80162 532752 994443 28414 855236 503983 213456 622322 577512 912457 241656 720105 324128 774460 479285 823522 68286 509035 92535 786207 234577 140470 182192 847008 546540 134335 910161 378216 565855 528870 695114 443062 582010 853644 34764 917261 946704 877970 545738 146476 570097 686234 995827 358037 192238 546244 538360 190664 537880 920946 2315 104776 783232 833668 327453 702996 169912 446875 154142 46190 712015 49048 136319 890647 942487 115485 248691 259856 763682 642378 729769 510426 231039 853761 527826 489141 886593 703522 707680 862116 720504 753230 310340 365990 500724 357030 349678 95485 260104 351180 143824 412266 288900 337248 180250 115181 395001 615940 764606 33993 780216 362583 210732 680061 181779 770109 547863 152971 913945 616468 875181 507188 527260 706559 573985 359417 620462 220062 13796 121435 922142 725586 556583 775514 714495 643389 91802 598840 196384 798847 880174 92215 360852 280313 502293 26838 187243 677069 941120 59862 798838 43163 675935 359418 650758 224965 638820 778881 479980 419946 311552 641454 694201 351533 188835 723264 223614 635375 426878 672121 984460 243471 828932 628377 963253 611376 387107 162659 892232 734323 490319 94934 72599 337463 312055 71796 582580 678431 189132 905790 756591 102796 283776 423157 117272 366749 33668 569690 943298 837749 887624 213422 910845 780277 166846 641073 281242 663036 631510 321061 691434 176749 307394 960331 468670 127529 411891 787556 346023 703631 257625 738609 280887 130498 501477 4658 812669 222051 323274 709481 963532 822801 255351 319576 628403 287663 38621 203902 684997 579997 943086 864028 196753 728381 239195 751541 712490 566447 103828 628297 554272 622804 238987 4120 979012 848590 423600 894244 979943 875069 573877 601582 547994 16706 464394 360695 31918
