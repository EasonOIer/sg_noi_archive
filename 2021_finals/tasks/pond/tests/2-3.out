1276176799141